src/main/java/Entities/MeleeEnemy.java

```diff
@@ -20,16 +20,17 @@ public MeleeEnemy(GamePanel gamePanel, int x, int y, int spawnX, int spawnY) {
         this.xEnemy = x + this.spawnX;
         this.yEnemy = y + this.spawnY;
     }
-    public void update() {
+/*    public void update() {
         if (!getHitBox().intersects(gamePanel.player.getHitBox())){
             enemyMovement();
         }
-    }
+    }*/
     public Rectangle getHitBox() {
         hitBox = new Rectangle(spawnX - 1280 + 4, spawnY - 720 + 4, 24, 24);
         return hitBox;
     }
 
+/*
     private void enemyMovement() { //In order to update current enemy location must update absXenemy.
         distance = Math.sqrt((Math.pow((gamePanel.player.getAbsXPlayer() - xEnemy - spawnX + 1896),2) + Math.pow((gamePanel.player.getAbsYPlayer() - yEnemy -spawnY + 1046), 2)));
         if (distance < 600 & distance > 110) {

```

Comment:
This is the code block that I was referring to in my previous comment. This chunk of commented out code should be removed.